docs/content/dependencies/metadata-storage.md

```diff
@@ -1,128 +1,19 @@
 ---
 layout: doc_page
 ---
+
 # Metadata Storage
 
 The Metadata Storage is an external dependency of Druid. Druid uses it to store
 various metadata about the system, but not to store the actual data. There are
 a number of tables used for various purposes described below.
 
-## Supported Metadata Storages
-
-The following metadata storage engines are supported:
-
-* Derby (default, but not suitable for production)
-* MySQL
-* PostgreSQL
-
-Even though Derby is the default, it works only if you have all Druid
-processes running on the same host, and should be used only for experimentation.
-For production, MySQL or PostgreSQL should be used.
-
-To choose the metadata storage type, set `druid.metadata.storage.type` to
-`mysql`, `postgres` or `derby`.
-Set other `druid.metadata.storage` configuration
-keywords as shown below to give Druid information about how to connect to
-the database.
-
-As discussed in [Including Extensions](../operations/including-extensions.html),
-there are two ways for giving Druid the extension files it needs for the
-database you are using.
-The first is to put the extension files in the classpath.  The second is to
-put the extension files in a subdirectory of
-`druid.extensions.directory` (by default `extensions` under the Druid working directory) and list the subdirectory name in
-`druid.extensions.loadList`.  The example properties below show the second
-way.
-
-## Setting up MySQL
-
-1. Install MySQL
-
-  Use your favorite package manager to install mysql, e.g.:
-  - on Ubuntu/Debian using apt `apt-get install mysql-server`
-  - on OS X, using [Homebrew](http://brew.sh/) `brew install mysql`
-
-  Alternatively, download and follow installation instructions for MySQL
-  Community Server here:
-  [http://dev.mysql.com/downloads/mysql/](http://dev.mysql.com/downloads/mysql/)
-
-2. Create a druid database and user
-
-  Connect to MySQL from the machine where it is installed.
-
-  ```bash
-  > mysql -u root
-  ```
-
-  Paste the following snippet into the mysql prompt:
-
-  ```sql
-  -- create a druid database, make sure to use utf8 as encoding
-  CREATE DATABASE druid DEFAULT CHARACTER SET utf8;
-
-  -- create a druid user, and grant it all permission on the database we just created
-  GRANT ALL ON druid.* TO 'druid'@'localhost' IDENTIFIED BY 'diurd';
-  ```
-
-3. Configure your Druid metadata storage extension:
-
-  Add the following parameters to your Druid configuration, replacing `<host>`
-  with the location (host name and port) of the database.
+Derby is the default metadata store for Druid. It is not suitable for production. Other, more production suitable metadata stores 
```

Comment:
I would name the production-viable MD stores by name here, as I think this page is a bit disorienting/scary if all it says is "WE ONLY HAVE DERBY BUT DERBY IS NO GOOD"